common/autoware_ad_api_msgs/srv/InterfaceVersion.srv

```diff
@@ -0,0 +1,6 @@
+---
+autoware_ad_api_msgs/ResponseStatus status
```

Comment:
For what do you need this status?
I guess we can guarantee this response always succeeds.
Then, we can simplify the interface that we must keep very stable. (= InterfaceVersion)

Reply:
I agree and removed it. https://github.com/autowarefoundation/autoware.universe/pull/704/commits/7f1b1e2c28f792c3cdc273f9cd45dc85bff9a0a7